src/main/scala/io/iohk/praos/domain/SlotInEpochCalculator.scala

```diff
@@ -0,0 +1,18 @@
+package io.iohk.praos.domain
+
+class SlotInEpochCalculator(epochLength: Int, slotDurationInMilliseconds: Int) {
```

Comment:
Minor detail: Maybe this class could be a `case class` as it isn't mutable?

Reply:
:+1: